packages/amplication-client/src/Entity/EntityFieldProperty.scss

```diff
@@ -0,0 +1,18 @@
+@import "../style/index.scss";
+
+.entity-field-list-item-property {
+  &__container {
+    color: var(--black100);
+    height: 20px;
+    border-radius: 35px;
```

Comment:
please create params at the top of the file and use the params instead of fixed values

Reply:
Replaced the values with variables and created a variable for border-radius as I could not find a variable with 35px as a value.